modules/tinymce/src/themes/silver/main/ts/ui/core/complex/AlignSelect.ts

```diff
@@ -22,6 +21,13 @@ const alignMenuItems = [
   { title: 'Justify', icon: 'align-justify', format: 'alignjustify' }
 ];
 
+const alignCommands = {
```

Comment:
Instead of declaring a new object for this, it'd be better if these were included in the existing `alignMenuItems` object. That has a few benefits, it keeps the configuration in one location and makes it harder to end up having something that throws an exception if a new format is added to `alignMenuItems` but not here.

Reply:
I should add that does mean you need to do an `Arr.find` which is slightly slower, given this only happens when you click it should be fine as we already use the `Arr.find` logic elsewhere here.

If you wanted to optimize it though, you could turn `alignMenuItems` and then use `Obj.mapToArray` when passing to `buildBasicStaticDataset`.

Reply:
Thanks for the suggestion. Have made the change to use `Arr.find`